packages/teleport/src/services/session/makeSession.ts

```diff
@@ -56,6 +57,7 @@ export default function makeSession(json): Session {
     clusterId: cluster_name,
     parties: parties ? parties.map(p => makeParticipant(p)) : [],
     addr: server_addr ? server_addr.replace(PORT_REGEX, '') : '',
+    participantModes: participantModes ? participantModes : [],
```

Comment:
```suggestion
    participantModes: participantModes || [],
```

Although in this case I think it's more likely that `participantModes` be `undefined` or `null` so maybe this is more appropriate...

```suggestion
    participantModes: participantModes ?? [],
```